src/citrine/resources/descriptors.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+from uuid import UUID
+
+from citrine._session import Session
+from citrine.informatics.descriptors import Descriptor
+from citrine.informatics.predictors import Predictor
+
+
+# Not a full Collection since CRUD operations are not valid for Descriptors
+class DescriptorMethods:
+    """[ALPHA] A set of methods returning descriptors which require connection to the backend."""
+
+    def __init__(self, project_id: UUID, session: Session):
+        self.project_id = project_id
+        self.session: Session = session
+
+    def from_predictor_responses(self, predictor: Predictor,
+                                 inputs: List[Descriptor]) -> List[Descriptor]:
+        """
+        [ALPHA] Get responses for a predictor, given a set of inputs.
+
+        Parameters
+        ----------
+        predictor The predictor whose available responses are to be computed.
```

Comment:
I don't think this part of the docstring is formatted quite correctly.  We use [numpy-style](https://numpydoc.readthedocs.io/en/latest/format.html#sections)